src/Microsoft.AspNetCore.Testing/ExceptionAssertions.cs

```diff
@@ -165,10 +173,9 @@ public static Task<ArgumentException> ThrowsArgumentNullOrEmptyAsync(Func<Task>
         /// <param name="testCode">A delegate to the code to be tested</param>
         /// <param name="paramName">The name of the parameter that should throw the exception</param>
         /// <returns>The exception that was thrown, when successful</returns>
-        /// <exception>Thrown when an exception was not thrown, or when an exception of the incorrect type is thrown</exception>
         public static ArgumentException ThrowsArgumentNullOrEmptyString(Action testCode, string paramName)
```

Comment:
I'm not sure of the current status but some follow-up may be worthwhile:
- Do both messages (`"Value cannot be null or empty."` and `"Value cannot be null or an empty string."`) exist in the wild? Are both variants of these assertions used too? Looks like something that should be narrowed down e.g. to just `"Value cannot be null or an empty string."`.
- These assertions will stop working as soon as our packages are localized and someone (even accidentally) writes a non-English test.